src/tests/integration/D3MAave.t.sol

```diff
@@ -1061,6 +1061,86 @@ contract D3MAaveTest is DSSTest {
         vat.frob(ilk, address(address(d3mAavePool)), address(this), address(this), 0, -100); // Some small amount of dai repaid
 
         // We should be able to close out the vault completely even though ink and art do not match
-        _setRelBorrowTarget(0);
+        // _setRelBorrowTarget(0);
+        d3mAavePlan.file("bar", 0);
+
+        d3mHub.exec(ilk);
+    }
+
+    function test_wind_partial_unwind_wind_debt_paid_back() public {
```

Comment:
We should clean up some leftover comments in this test.

Reply:
If you meant the `vat.sin` ones were commented due stack too deep. But seems Chris did actually replicate some variable cleaning in the integration tests and they can now be enabled (which is done now + adding `vice` checks)

Reply:
ok.